packages/graph/hash_graph/tests/integration/postgres/entity_type.rs

```diff
@@ -12,7 +14,7 @@ fn insert() {
             [data_type::TEXT_V1],
             [property_type::NAME_V1],
             [link_type::FRIEND_OF_V1],
-            [],
+            [entity_type::ORGANIZATION_V1],
```

Comment:
This doesn't seem to be needed either, the tests still pass without. Was the intention to have another `PERSON` version?
```suggestion
            [],
```

Reply:
This was a copy/paste error wrongly editing the entity type tests. Thanks for flagging!